webui/src/store/modules/okr.js

```diff
@@ -175,7 +178,16 @@ export default {
                 user.state.selectedSubject.id,
                 user.state.me.id,
                 data => commit('OBJECTIVES_COMPLETE', data),
-                err => commit('OBJECTIVES_FAILED', err)
+                err => {
+                    if (err.statusCode == 404 && err.code == 30108) {
+                        // The OneDriveForBusiness for this user account cannot be retrieved
+                        state.invalidOneDriveForBusinessLicense = true;
+                        // No objectives available for such users
+                        commit('OBJECTIVES_COMPLETE', [])
```

Comment:
The OBJECTIVES_COMPLETE mutation will immediately set invalidOneDriveForBusinessLicense back to false